docs/interfaces/mxv/leds.md

```diff
@@ -0,0 +1,32 @@
+---
+title: LEDs (MX-V)
+tags:
+  - LED
+  - MX-V
+---
+
+## Overview
+
+This platform has red and greed LEDs.
+
+All leds can be used as you like and they are just named.
+
+**Warning:** GNSS led can be lit when powered and if it not turn off in 1 sec the main cpu has been hanged, the co-cpu will restart the system and if it continues the unit has failed to boot u-boot.
+
+Start up the func led red is lit when enter u-boot state and off in linux.
+
+The power led is controlled buy the co-cpu and the func led green is setup from device tree with heartbeat trigger and will blink if the system is running.
+The power led is lit green when co-cpu is runing and somtimes the co-cpu lit the red led short it is currently some debug prints and if it is solid lit it has an error.
```

Comment:
Do you mean that the short periods of red are normal and solid red is error, or is short red an error as well?

Reply:
solid red are error but it acts like a heartbet, i am not sure why part of the code use it like that but it is the current state.

Reply:
@mattiasbus  do you have any sugesstion to what the red led does in the system?

Reply:
I cannot remember. Have to look in the code.

Reply:
we leave it as it is